### ISSUE TITLE
Version1.0 incorrect versioning label (remains 0.1)

### ISSUE THREAD

**Issue by AustenWallis** [sirocco-rt/sirocco#1133]
To note:
I've been fresh installing Sirocco for HPCs and noticed the main-branch/1.0 tag has the source version number still set as 0.1. 
![Screenshot 2024-12-31 at 12 59 35](https://github.com/user-attachments/assets/0361a4a5-aad4-4147-b952-a7372c6d2edf)

The result when looking at a sirocco runs log files is "Python version 0.1" ... "python is running with X processes". 
![Screenshot 2024-12-31 at 13 27 03](https://github.com/user-attachments/assets/96f5b9ef-e2df-4c28-9d80-362d1a482569)

I imagine this will be confusing for new users as this is likely the first log info they'll read when doing their first run. 
This will likely be fixed when a new version number is created (1.1). 
Updating the code so PYTHON -> Sirocco in the log will be useful too when the next new version is introduced. 

But opening this issue now as a notice until resolved in next iteration

**Comment by jhmatthews** [comment]
Good catch Austen. The version thing can be changed now, although it's annoying I forgot to do this for the 1.0 release. 

As for the Python string - well-spotted. It seems I actually missed a load of text about Python in the source code which should be fixed. I think I got all the python and PYTHON cases, but not Python. Silly of me.
